R/uvhydrograph-data.R

```diff
@@ -409,8 +414,10 @@ hasUpchainSeries <- function(reportObject) {
 #' @param isDischarge whether or not the primary series is discharge
 #' @return the lims object that to be applied to the primary plot
 calculatePrimaryLims <- function(primarySeriesList, isDischarge) {
-  if(!is.null(primarySeriesList[['corrected']])){
+  if(!isEmptyOrBlank(primarySeriesList[['corrected']][['points']][['value']])){
     lims <- calculateLims(primarySeriesList[['corrected']][['points']])
+  } else if(primarySeriesList[['useEstimated']]){
+    lims <- calculateLims(primarySeriesList[['estimated']][['points']])
   } else {
     lims <- calculateLims(primarySeriesList[['uncorrected']][['points']])
   }
```

Comment:
I'm not sure.  I left it in because it was there already. However, based on other observations that Laura and Chuck made when testing my first attempt, it appears that the y-axis limit calculation will need to be refactored to account for estimated points anyway.